pkg/apis/camel/v1/common_types.go

```diff
@@ -48,12 +47,17 @@ const (
 	// BuildStrategyPod performs the build in a `Pod` (will schedule a new builder ephemeral `Pod` which will take care of the build action).
 	// This strategy has the limitation that every build will have to download all the dependencies required by the Maven build.
 	BuildStrategyPod BuildStrategy = "pod"
+	// BuildStrategyType is the type of strategy that should be used to perform the build.
```

Comment:
"BuildStrategyMvnd performs the build using Apache Maven Daemon, that's started in the operator Pod."